src/components/SideBar.tsx

```diff
@@ -55,7 +55,7 @@ export default function SideBar({ mobile, closeSidebar }: { mobile?: boolean; cl
     })
 
     const importToMetamask = async () => {
-        const allTokens = []
+        const allTokens: Array<any> = []
```

Comment:
Array<T> is obsolete notation, we prefer to use T[]

```suggestion
        const allTokens: any[] = []
```

Reply:
done